packages/visual-editor/src/components/Dropzone/EditorBlock.tsx

```diff
@@ -87,6 +95,36 @@ export const EditorBlock: React.FC<EditorBlockProps> = ({
     });
   };
 
+  if (isComponentMissing || !definition || !elementToRender) {
+    const renderMissingComponentPlacehoder = () => (
```

Comment:
I think this div and related styles could live in the SelectedComponentPlaceholder itself, and just the pass the children into it as a prop.